.eslintrc

```diff
@@ -1,14 +1,16 @@
 {
   "root": true,
   "extends": "airbnb-base",
+  "env": {
+    "jest": true,
+  },
   "rules": {
     "max-len": 0,
     "no-console": 0,
   },
   "overrides": [{
     "files": ["test/**/*.js"],
     "rules": {
-      "no-undef": "off",
       "global-require": "off",
       "no-unused-vars": "off",
```

Comment:
Do you also need to disable this rule? I'm not sure when you'd actually need unused vars in your tests.

Reply:
looks like unused variables are used to stub arguments for traversal calls. They're used pretty extensively in the tests: 
https://github.com/airbnb/react-component-variations/blob/f7ef780d19d79ee3a068381a864ceb9c6231cfa4/test/traversal/forEachDescriptor.js#L43

Reply:
Ah I see! I think disabling this rule is okay for now, but agree that a better fix would be to either use eslint-disable-next-line where necessary or modify the code so it uses the args in the body.